src/components/mainMenu.js

```diff
@@ -0,0 +1,13 @@
+const mainMenu = document
+  .querySelector('.swquiz-mainmenu')
+  .querySelectorAll('p');
+
+mainMenu.forEach((element) => {
+  element.addEventListener('click', function () {
+    mainMenu.forEach((item) => item.classList.remove('selected'));
+
+    this.classList.add('selected');
+  });
+});
+
+export default mainMenu;
```

Comment:
Same stylowanie wydaje mi sie ok. Jesli chodzi o te funkcje to wydaje mi sie ze powinna jeszcze wypluwac z siebie jakas zmienna, ktora ma potem trafic do funkcji, ktora ja napisalam. Tj musi wypluc stringa ("people"/"vehicle"/"starships") w zaleznosci od tego co uzytkownik kliknie, zeby wybrac w jakim mode gramy. Same elementy HTML musialyby przechowywac te values np. jako atrybut i przekazywac dalej w chwili klikniecia.

Reply:
No właśnie ja tego do końca nie widzę i nie potrafię sobie wyobrazić. Czy dołączenie data-test attributes do elementów People, Vehicles, Starships by się nadawało?

Reply:
Mysle, ze atrybut data bylby ok. Mozesz tez uzyc value, ale to wtedy chyba musisz zmienic <p> na cos innego np. button ( https://www.geeksforgeeks.org/html-value-attribute/ ) i pobierac przez e.target.value ( https://dillionmegida.com/p/understanding-event-target-and-use-cases/ ) :)